lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -53,9 +57,20 @@ def get(self, request, course_key_string):
         course_key = CourseKey.from_string(course_key_string)
         course_tools = CourseToolsPluginManager.get_enabled_course_tools(request, course_key)
 
+        course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=False)
+        blocks = get_course_date_blocks(course, request.user, request, num_assignments=1, include_past_dates=False)
```

Comment:
The `include_past_dates` parameter defaults to `False` so you can actually just leave it out.